src/Draft/List/FlatList.vue

```diff
@@ -1,33 +1,32 @@
 <template>
-  <ul class="my-8 space-y-8 bg-transparent">
-    <li v-for="(_listItem, index) in listItems" :key="index">
+  <ul
+    v-for="(_listItem, index) in listItems"
```

Comment:
interesting semantics here. We are changing from one `ul` containing n `li` list items to n `ul` items each containing 1 `li` item. Is this necessary?

Reply:
I see the point. Good catch.